packages/amplify-migration-tests/tsconfig.tests.json

```diff
@@ -7,7 +7,6 @@
     "outDir": "lib",
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
-    "esModuleInterop": true,
```

Comment:
what did this do? Why are we removing it

Reply:
This was me playing with various configurations in the attempt to fix this issue: ![image](https://user-images.githubusercontent.com/85319655/236901413-77c92624-edde-4c6b-bdca-7352bd7eaa75.png)

If you feel strong about this change, I can revert it in this PR as there is a followup PR for migration and E2E tests: https://github.com/aws-amplify/amplify-cli/pull/12592